docker/docker/pom.xml

```diff
@@ -97,6 +97,11 @@
             <artifactId>snakeyaml</artifactId>
         </dependency>
 
+		<dependency>
+			<groupId>org.eclipse.jgit</groupId>
+			<artifactId>org.eclipse.jgit</artifactId>
```

Comment:
I would add this dependency as provided and add in documentation that if you want to use `context` with git repository you need to add the jgit dependency. I said this because I think that almost all users will not use git feature, maybe `context` but without git. Then we are adding a dependency with all transitive dependencies to all users.